### DISCUSSION TITLE
New Dialogs for Switches... ouch

### DISCUSSION THREAD

**Discussion by z3liff** [home-assistant/frontend#15682] [Dashboard]
This is largely a preference discussion, but the new switch dialogs feel pretty close to useless.  It was nice having relevant information and options readily available right at the top-level of the switch dialogs.  Now, it's numerous clicks to accomplish what _was_ a single click.  Want to view state?  more clicks! want to view options such as effects or color pickers? more clicks!

If I wanted to just click a button to toggle, I would do that _without_ bringing up a dialog.  It literally relegates the intent of the dialog to a secondary aspect of the dialog, for no reason other than "big button syndrome".

I would propose an option to revert back to the "information dense" dialogs.   Am I alone in this thought and preference?

**Comment by ildar170975** [comment]
Absolutely agree.
The new design may only be useful on mobile phones, but not on tablets or PCs. 
And same design is currently added for input_boolean.
Were these changes properly discussed in HA team?!

**Comment by z3liff** [comment]
appears to be another thread started here as well:
https://github.com/home-assistant/frontend/discussions/16115
